src/bip44.c

```diff
@@ -118,6 +118,17 @@ bool bip44_hasPoolColdKeyPrefix(const bip44_path_t* pathSpec)
 #undef CHECK
 }
 
+// /1694'/1815'
+bool bip44_hasGovernanceVotingKeyPrefix(const bip44_path_t* pathSpec)
+{
+#define CHECK(cond) if (!(cond)) return false
+	CHECK(pathSpec->length > BIP44_I_COIN_TYPE);
+	CHECK(pathSpec->path[BIP44_I_PURPOSE] == (PURPOSE_GOVERNANCE_VOTING_KEY | HARDENED_BIP32));
+	CHECK(pathSpec->path[BIP44_I_COIN_TYPE] == (ADA_COIN_TYPE | HARDENED_BIP32));
```

Comment:
Maybe you could use the `harden` and `isHardened` functions here and in `bip44_isGovernanceVotingKeyPath`?